web/sites/default/config/field.field.group.bureau_office.field_group_path.yml

```diff
@@ -0,0 +1,25 @@
+uuid: b430cde1-a011-4728-a181-7b683dd5e538
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.group.field_group_path
+    - group.type.bureau_office
+  module:
+    - custom_add_another
```

Comment:
I wonder why this was a dependency. Doesn't seem like this field would have an "add another" function. 🤔

Reply:
Not sure how it got in there. This is a singleton field.